hydroshare/hydroshare
# Converting Organization Field to Tag Input

Resolves #2309

To test:
1. Check out the branch
2. Navigate to the user profile page and click the pencil to activate edit mode
3. Note the edit field for organization
4. When adding an organization there should be a drop down containing the same university data set as the hs_dictionary app in the admin (http://192.168.56.101:8000/admin/hs_dictionary/university/ or similar)
5. It should allow for a maximum of two organizations

Another use case:

Neal, there are a lot of moving parts here but we're testing both the dictionary system as well as a new registration field. Here goes:

1. Check out 2283-normalizing-university-data-set
2. ./hsctl rebuild --db
3. Open http://192.168.56.102:8000/sign-up
4. Note the new field
5. Fill out the form and ensure that the tag input works correctly with the tab key
6. Enter two organizations - one university from the autocomplete and a second that you know is not in the DB (for testing later)
7. Complete registration
8. Use your superuser account to activate your account (or activate via email if it is sent)
9. Log in and navigate to your user page i.e. http://192.168.56.102:8000/user/941/
10. Click the pencil to edit and ensure that the organization field carried through to the profile
11. Navigate to the admin panel and check the "Uncategorized Terms" table to make sure your second organization made it there
